src/common/models/Projections.ts

```diff
@@ -95,6 +95,17 @@ export class Projections {
     };
   }
 
+  hasMetric(metric: Metric): boolean {
+    if (metric === Metric.FUTURE_PROJECTIONS) {
+      return (
+        this.baseline?.hasHospitalProjections &&
+        this.projected.hasHospitalProjections
+      );
+    } else {
+      return this.getMetricValue(metric) !== null;
+    }
+  }
```

Comment:
New helper to clean up some code now that future projections may not be present.